src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/HttpRequestParser.cs

```diff
@@ -448,6 +448,10 @@ private void ConsumeCurrentTokenInto(HttpSyntaxNode node)
                         node = new HttpUrlNode(_sourceText, _syntaxTree);
                         node.Add(ParseEmbeddedExpression());
                     }
+                    else if (CurrentToken is { Kind: HttpTokenKind.Punctuation })
+                    {
+                        node = new HttpUrlNode(_sourceText, _syntaxTree);
```

Comment:
Why is it that we only call ParseLeadingWhitespaceAndComments in the case where the current token is a word (line 444)? Should we also call that here (as well as in the embedded expression case above before line 449)?

Reply:
In other words, do we drop leading trivia / comments in the two cases where the call is currently missing?

Reply:
(Thanks for fixing - It is interesting that the missing trivia parsing was not caught by the tests since iirc all tests test round-tripping...)